lib/rules/no-test-import-export.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @fileOverview no importing of test files and no exporting in test files
+ */
+
+'use strict';
+
+const NO_IMPORT_MESSAGE
+  = 'Do not import "-test.js" file in a test file. This will cause the module and tests from the imported file to be executed again.';
```

Comment:
"Do not import from a test file (a file ending in "-test.js") in another test file. Doing so will cause the module and tests from the imported file to be executed again."